spamhandling.py

```diff
@@ -103,8 +101,7 @@ def handle_spam(title, body, poster, site, post_url, poster_url, post_id, reason
             username = poster.strip()
             user_link = poster_url
 
-        t_metasmoke = Thread(target=Metasmoke.send_stats_on_post,
```

Comment:
Looks like you went a little too far. :) We had two lines for these and the ones in ws.py